.github/workflows/release-plz.yml

```diff
@@ -0,0 +1,60 @@
+name: Release-plz
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+
+  # Release unpublished packages.
+  release-plz-release:
+    name: Release-plz release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Run release-plz
+        uses: release-plz/action@v0.5
+        with:
+          command: release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
default github token [should suffice for now ](https://release-plz.dev/docs/github/token):
> GitHub Actions using the default [GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) cannot trigger other [workflow](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) runs, i.e. cannot start new GitHub Actions jobs.